local-cli/server/runServer.js

```diff
@@ -83,7 +83,7 @@ function getPackagerServer(args, config) {
       'bmp', 'gif', 'jpg', 'jpeg', 'png', 'psd', 'svg', 'webp', // Image formats
       'm4v', 'mov', 'mp4', 'mpeg', 'mpg', 'webm', // Video formats
       'aac', 'aiff', 'caf', 'm4a', 'mp3', 'wav', // Audio formats
-      'html', // Document formats
+      'html', 'pdf', // Document formats
```

Comment:
I don't think it should be in `rn-cli.config`. It's not a configuration option. It's just the list of what's supported by RN. If someone adds a random extension there it won't work, since the library doesn't support it.